python/Ganga/Core/GangaRepository/GangaRepositoryXML.py

```diff
@@ -496,6 +496,8 @@ def update_index(self, this_id=None, verbose=False, firstRun=False):
                     ## we can't reasonably write all possible exceptions here!
                     logger.debug("update_index: Failed to load id %i: %s" % (this_id, str(x)))
                     summary.append((this_id, str(x)))
+                #finally:
+                #    pass
```

Comment:
It's used when commenting out the exception handler because it's hiding problems rather than re-indent everything else for the sake of it